app/src/main/assets/i18n/Scribe-i18n/values/en-US/string.xml

```diff
@@ -27,7 +27,7 @@
     <string name="app.about.feedback.rate_conjugate">Rate Scribe Conjugate</string>
     <string name="app.about.feedback.rate_scribe">Rate Scribe</string>
     <string name="app.about.feedback.title">Feedback and support</string>
-    <string name="app.about.feedback.version">Version</string>
+    <string name="app.about.feedback.version">Version %1$s</string>
```

Comment:
Should be updated in Scribe-i18n repo also

Reply:
Could you remove these changes for now where the assets folder values are being changed? It would mess up the git subtree.  @andrewtavis Should we make the text for now be taken from strings.xml which Android has given by default until the string is changed?